sdks/go/pkg/beam/runners/dataflow/translate.go

```diff
@@ -135,6 +154,127 @@ func translate(edges []*graph.MultiEdge) ([]*df.Step, error) {
 	return steps, nil
 }
 
+func expandCoGBK(nodes map[int]*outputReference, edge *graph.MultiEdge) ([]*df.Step, error) {
+	// TODO(BEAM-490): replace once CoGBK is a primitive. For now, we have to translate
+	// CoGBK with multiple PCollections as described in graphx/cogbk.go.
```

Comment:
Is it worth including a reference comment here to the ASCII art picture that explains the steps happening here?